Changelog for 0.34.0

SimonBiggs opened the pull request:
Prep for 0.34.0 release.

SimonBiggs commented on CHANGELOG.md:
```diff
@@ -8,12 +8,155 @@ This project adheres to
+* A new CLI utility that propagates a range of files that depend upon each
+  other files within the repo are "propagated" by calling
+  `poetry run pymedphys dev propagate`. The files created/updated by this
+  command are `lib/pymedphys/_version.py`, the 'extras' field within
```
Mention the extras field last

SimonBiggs commented on CHANGELOG.md:
```diff
@@ -8,12 +8,155 @@ This project adheres to
+* Removed `pymedphys experimental gui`.
+
+
+### Developer facing changes
```
```suggestion
### Developer facing API changes
```

SimonBiggs commented on CHANGELOG.md:
```diff
@@ -8,12 +8,155 @@ This project adheres to
+
+### Developer facing changes
+
+* The pymedphys library directory has moved to `lib/pymedphys`.
```
```suggestion
* The pymedphys library directory has moved to `lib/pymedphys`.
  After this move you will need to rerun `poetry install -E dev`
```

sjswerdloff commented on CHANGELOG.md:
```diff
@@ -8,12 +8,156 @@ This project adheres to
+#### New features
+
+* Created an online PyMedPhys GUI. It is accessible from
+  https://app.pymedphys.com. This is in its early stages and most parts of the
```
might want to say the hosted version is intended for demonstration purposes only, and isn't appropriate for "daily use" (better that the facility host locally).
You could put in a query to see if there is interest in a hosted service.

SimonBiggs replied:
Yup, I agree. And I should also add the warning that when using the demo sensitive information should not be uploaded. If the user wants to utilise the tools with sensitive information they should install it on their own machine.

sjswerdloff commented on CHANGELOG.md:
```diff
@@ -8,12 +8,156 @@ This project adheres to
+  `pyproject.toml`, `requirements.txt` and `requirements-dev.txt`,
+  `lib/pymedphys/.pylintrc`, `lib/pymedphys/docs/README.rst`, and
+  `lib/pymedphys/docs/CHANGELOG.md`.
+* `pymedphys dev tests` now allow optional simpler flags. `--run-only-slow` can
```
...tests` _includes simplified flags_

SimonBiggs commented:
For an insight into what I'm doing here, I'm using this PR facility to be able to step through all the changes since the last release and mark them as "viewed":

![image](https://user-images.githubusercontent.com/6559099/99031383-0c4e9f00-25cb-11eb-99fc-4a4df20ce30d.png)

That way I can write up the changelog in a systematic way.
